dotnet/src/webdriver/IWebElement.cs

```diff
@@ -170,7 +170,7 @@ public interface IWebElement : ISearchContext
         /// </list>
         /// </remarks>
         /// <exception cref="StaleElementReferenceException">Thrown when the target element is no longer valid in the document DOM.</exception>
-        [Obsolete("Use GetDomAttribute(string attributeName) or GetDomProperty(string propertyName")]
+        [Obsolete("Use GetDomAttribute(string attributeName) or GetDomProperty(string propertyName). GetAttribute() is schedule for removal in Selenium v6"]
```

Comment:
`GetAttribute(string attributeName) will be removed in Selenium 6.` - full signature and typo.

Reply:
Done